src/ClientData/include/ClientData/MockScopeIdProvider.h

```diff
@@ -0,0 +1,30 @@
+
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_CLIENT_DATA_API_MOCK_SCOPE_ID_PROVIDER_H_
```

Comment:
Why not ORBIT_CLIENT_DATA_MOCK_SCOPE_ID_PROVIDER_H_ ?

Reply:
Because it was copied from another incorrectly written one :) fixed.